storage/integration_test.go

```diff
@@ -348,12 +348,20 @@ func TestIntegration_DetectDirectConnectivity(t *testing.T) {
 			region := v.AsString()
 			newBucketName := prefix + uidSpace.New()
 			newBucket := client.Bucket(newBucketName)
-			h.mustCreate(newBucket, testutil.ProjID(), &BucketAttrs{Location: region, LocationType: "region"})
+			h.mustCreate(newBucket, testutil.ProjID(), &BucketAttrs{Location: region})
 			defer h.mustDeleteBucket(newBucket)
 			err := CheckDirectConnectivitySupported(ctx, newBucketName)
 			if err != nil {
 				t.Fatalf("CheckDirectConnectivitySupported: %v", err)
 			}
+			newMRBucketName := prefix + "-mr-" + uidSpace.New()
+			newMRBucket := client.Bucket(newMRBucketName)
+			h.mustCreate(newMRBucket, testutil.ProjID(), &BucketAttrs{Location: "US"})
+			defer h.mustDeleteBucket(newMRBucket)
+			err = CheckDirectConnectivitySupported(ctx, newMRBucketName)
+			if err == nil {
+				t.Fatalf("CheckDirectConnectivitySupported: error expected, got nil")
+			}
```

Comment:
So this covers the multi-regional case. Should we add a few more cases? I think we should cover:

1. single region, same region as VM (expect success)
2. single region, diff region from VM (fail)

Same thing with dual region and multiregion.

We could probably refactor this as a table driven test to make it clearer.

Reply:
make sense! i'll work that next. thanks